src/content/doc-surrealql/statements/define/field.mdx

```diff
@@ -196,6 +196,64 @@ DEFINE FIELD locked ON TABLE user TYPE bool
 DEFAULT false;
 ```
 
+### Using the `DEFAULT` and `ALWAYS` clause
+
+<Since v="v2.2.0" />
+
+In addition to the `DEFAULT` clause, you can use the `DEFAULT ALWAYS` clause to set a default value for a field. The `ALWAYS` keyword indicates that the `DEFAULT` clause is used not only on `CREATE`, but also on `UPDATE` if the value is empty (NONE).
+
+
+```surql
+DEFINE TABLE product SCHEMAFULL;
+-- Set a default value of 123.456 for the primary field
+DEFINE FIELD primary ON product TYPE number DEFAULT ALWAYS 123.456;
+```
+
+With the above definition, the `primary` field will be set to `123.456` when a new `product` is created without a value for the `primary` field or with a value of `NONE`, and when an existing `product` is updated if the value is specified the result will be the new value. 
+
+In the case of `NULL` or a mismatching type, an error will be returned.
+
+```surql
+-- This will return an error
+CREATE product:test SET primary = NULL;
+
+-- result 
+Found NULL for field `primary`, with record `product:test`, but expected a number
+```
+
+On the other hand if the value is defined or updated to a number, the value will be set to the default value of `123.456`.
```

Comment:
```suggestion
On the other hand, if a valid number is provided during creation or update, that number will be used instead of the default value. In this case, `123.456`.
```